remoteworkitem/remoteworkitem.go

```diff
@@ -74,7 +74,7 @@ var RemoteWorkItemKeyMaps = map[string]RemoteWorkItemMap{
 	},
 	ProviderJira: {
 		AttributeMapper{AttributeExpression(JiraTitle), StringConverter{}}:                                      remoteTitle,
-		AttributeMapper{AttributeExpression(JiraBody), MarkupConverter{markup: rendering.SystemMarkupJiraWiki}}: remoteDescription,
+		AttributeMapper{AttributeExpression(JiraBody), MarkupConverter{markup: rendering.SystemMarkupMarkdown}}: remoteDescription,
```

Comment:
Can you please explain for why you did this change? @xcoulon do you remember if the JIRA syntax was ever supported?

Reply:
@kwk 
We don't support `SystemMarkupJiraWiki` now ->
https://github.com/fabric8-services/fabric8-wit/blob/master/workitem/field_test_data.go#L244

Reply:
I would suggest not to support Jira markup. Let's treat it as plain text.

Reply:
@kwk Based on the discussion, we will address this in a separate PR. I will create an issue for the same.

Reply:
Please create and reference the issue here @DhritiShikhar

Reply:
https://github.com/fabric8-services/fabric8-wit/issues/2365